### ISSUE TITLE
Save user data

### ISSUE THREAD

**Issue by Bergiu** [TheWildBear/bigDataTelgrambot#31]
Save some more information about users, for example there username, there real id
This is needed to provide an analysis per user and see which user it who.

**Comment by TheWildBear** [comment]
I'll need to take a look into it what could be collected. It of course has to be optin.

**Comment by Bergiu** [comment]
Yes, you can make these levels you talked about in the telegram chat:
1. Don't save me
2. Only count my messages without text
    - this would save the chat id, a hashed id of the user, and the date of the message
3. Only save my messages without my data
    - this would save the chat id, a hashed id of the user, the date of the message and the text
4. Save my userprofile
    - this would save the chat id, the real id of the user, the date of the message, the text and the user profile information
